apps/src/templates/curriculumCatalog/CurriculumCatalogCard.jsx

```diff
@@ -22,7 +23,8 @@ const CurriculumCatalogCard = ({
   imageSrc,
   subjects,
   topics,
-  isTranslated
+  isTranslated,
+  language
```

Comment:
are you aware of any place in the catalog we will need the language for anything besides determining whether the page is in english? if not, my recommendation would be "YAGNI" and just assume isEnglish will be passed into this component.